packages/orbit-components/src/InputSelect/__tests__/index.test.tsx

```diff
@@ -216,6 +216,50 @@ describe("InputSelect", () => {
     expect(screen.getAllByRole("option")).toHaveLength(1 + 2 + 1 + 4); // (1 previously selected, 2 asian, 1 american, 4 all)
   });
 
+  it("when readOnly is true", async () => {
+    render(
+      <InputSelect
+        id={id}
+        label={label}
+        options={options}
+        name={name}
+        defaultSelected={jetLiOption}
+        readOnly
+      />,
+    );
+
+    await user.tab();
+
+    const input = screen.getByRole("textbox");
+    const dropdown = screen.queryByRole("listbox");
+
+    expect(input).toHaveValue(jetLiOption.title);
+
+    // Check if dropdown is not shown on a click
```

Comment:
Before this, you did a `user.tab()`, so the input would be focused, the open even to be triggered already. So I would assert that as well, before clicking